setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='gp-multistate-kernel',
-    version='0.0.9',
+    version='0.1.0',
```

Comment:
Could you please use `0.0.10` here instead of `0.1.0` ?

Reply:
OK